_episodes/30.md

```diff
@@ -0,0 +1,244 @@
+</p>
+
+The early architecture for the analytics platform followed a more traditional
+approach using Spark and Fivetran to ingest data into Redshift and had various
+pipelines to update the data for downstream consumption.
+
+This centralized workflow made communication across data entity management much
+simpler as it all exists on the same team. However, this made scaling cause 
+increased backlogs which delayed analysis and deployments. It also made it 
+difficult to handle different use cases like realtime and historical use cases.
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/30/talkdesk-architecture.png"/><br/>
+</p>
+
+There is a wide gap between analytics use cases and transactional use cases and 
+many in between. Live data typically feeds into stateful databases that update 
+as data arrives. To analyze this data in motion, you need a realtime 
+database to process this data as it is inserted. Historical data exists to keep 
+a backup of multiple copies of different states over time. This enables analysis
+of trends over longer periods of time versus right now. How can we realize a
+robust architecture that satisfies analyzing live data that gets the latest 
+changes as they arrive to OLTP databases and merge it with the analytics uses?
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/30/olap-oltp.png"/><br/>
+</p>
+
+To do this, Talkdesk integrated Trino into their workflow to read data across
+both live and historic data and merge them. This enables reading from live data
+feeding into their stateful data stores, and reads across historic data stores
+to produce data in the form needed to support Talkdesk products.
+
+<p align="center">
+<img align="center" width="90%" height="100%" src="/assets/episode/30/talkdesk-architecture-2.0.png"/><br/>
+</p>
+
+Trino uses also are used to hide the complexity of the data platform and allows
+merging data across mulitple relational and object stores.
+
+<p align="center">
+<img align="center" width="60%" height="100%" src="/assets/episode/30/talkdesk-architecture-2.0-external.png"/><br/>
+</p>
+
+### Why dbt?
+
+In [episode 21](/episodes/21.html) we discussed using dbt + Trino in detail. As
+we mentioned there:
+
+> dbt is a transformation workflow tool that lets teams quickly and 
+> collaboratively deploy analytics code, following software engineering best 
+> practices like modularity, CI/CD, testing, and documentation. It enables 
+> anyone who knows SQL to build production-grade data pipelines. 
+
+You can achieve modularity and repeatable testable units of processing by 
+defining various models and definitions to the data pipelines. For example:
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/30/dbt-definition.png"/><br/>
+</p>
+
+Using the definitions above, Talkdesk engineers were able to consolidate all
```

Comment:
seems redundant?

```suggestion
Using the definitions above Talkdesk engineers were able to consolidate all
```

Reply:
Don't know why myself, but this feels correct. If I read it aloud there's a pause there.